src/Components/Progress.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Progress.css';
+
+export default function Progress() {
+  return (
+    <div className="current-progress">
+      <div className="progress-tracker">
+        <div className="progress">
+          <div className="progress-status" />
+        </div>
+        <div className="statistics">
+          <p>80%</p>
+          <p>Completed</p>
+        </div>
+      </div>
+      <div className="vertical-pline" />
+      <div className="chapter-status">
+        <div className="chapter-number">
+          <p>CURRENT CHAPTER</p>
+          <p>Chapter 20</p>
+          <button type="button" className="update-progress">
+            UPDATE PROGRESS
+          </button>
+        </div>
```

Comment:
-  to match the [required design](https://pedantic-beaver-cb923b.netlify.app/#/) please fix this issues 🙏  : 
- [ ] Update the color for the 'current chapter' title as shown in the design
- [ ] add some space between chapter 20 & the button below it
- [ ] increase the size of the percentage and change the color of 'completed' below it


Required design : 

![image](https://user-images.githubusercontent.com/89282221/215289453-c531ad1c-5558-4e34-80a1-a1156f88bb3b.png)

your design : 
![image](https://user-images.githubusercontent.com/89282221/215289476-7e53deb3-530a-4191-8c9d-cead92f19891.png)